### PR TITLE
Allow to support newest grape entity

### PR DESCRIPTION
**Pull request by Bugagazavr**

### CODE REVIEW

File: grape-swagger-entity.gemspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.email         = ['kirik910@gmail.com']
 
   s.summary       = 'Grape swagger adapter to support grape-entity object parsing'
-  s.homepage      = 'https://github.com/Bugagazavr/grape-swagger-entity'
+  s.homepage      = 'https://github.com/ruby-grape/grape-swagger-entity'
```
Comment (Bugagazavr) [inline grape-swagger-entity.gemspec]: I think i can do this in separated PR, if this is a problem

**Comment by Bugagazavr** [comment]
@dblock Looks like danger requires some updates

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #### Fixes
 
 * [#8](https://github.com/ruby-grape/grape-swagger-entity/pull/8): Generate enum property if values key is passed in documentation - [@lordnibbler](https://github.com/lordnibbler).
+* [#15](https://github.com/ruby-grape/grape-swagger-entity/pull/15): Allow to support grape entity 0.6.x and later - [@Bugagazavr](https://github.com/Bugagazavr).
```
Comment (dblock) [inline CHANGELOG.md]: Should just say "Support ...", "Allow" is unncessary.

**Comment by dblock** [comment]
Would you mind fixing the danger stuff too? It should be:

```ruby
gem 'ruby-grape-danger', '~> 0.1.0', require: false
```

And Dangerfile should just have

```ruby
danger.import_dangerfile(gem: 'ruby-grape-danger')
```

**Comment by Bugagazavr** [comment]
Ok i'l update `.travis.yml`, because danger requires `ruby 2.x`

**Comment by dblock** [comment]
This is perfect. Want to make a release?

**Comment by Bugagazavr** [comment]
Yep

**Comment by Bugagazavr** [comment]
done

**Pull request merged**